brayns/io/ProteinLoader.cpp

```diff
@@ -325,9 +335,19 @@ bool ProteinLoader::isSupported(const std::string& filename BRAYNS_UNUSED,
 }
 
 ModelDescriptorPtr ProteinLoader::importFromFile(
-    const std::string& fileName, const LoaderProgress&, const size_t index,
+    const std::string& fileName, const LoaderProgress&,
+    const PropertyMap& propertiesTmp, const size_t index,
     const size_t defaultMaterialId BRAYNS_UNUSED) const
 {
+    PropertyMap properties = getProperties();
+    properties.fillPropertyMap(propertiesTmp);
```

Comment:
I added a comment about this. The issue is when the property map is parsed it guesses the property types. This is because there is ambiguity when parsing numbers since they can be either floating point or int.